jobs-builder/jobs/include/os2node.yaml.inc

```diff
@@ -0,0 +1,21 @@
+{#
+# Copyright (c) 2020 Red Hat, Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Convert OS name to node label string.
+#}
+{% if os == "centos8" -%}
+centos8_azure
+{% elif os == "debian10" -%}
+debian10_azure
+{% elif os == "fedora32" -%}
+fedora32_azure
```

Comment:
F32 goes EOL in a few months - could this be changed to F33?

Reply:
As we add new jobs I plan to update the os2node.yaml.inc file. For now, it is good enough for the qemu and kernel jobs.